### PR TITLE
doc: fix errors in "search" page and use relative address for releases.json

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
```
> ceph-dashboard@0.0.0 test:config /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/dashboard/frontend
> if [ ! -e 'src/unit-test-configuration.ts' ]; then cp 'src/unit-test-configuration.ts.sample' 'src/unit-test-configuration.ts'; fi

...........................................................................................................
  ● PoolDetailsComponent › Pool details tabset › pool data filtering is called

    HttpErrorResponse: Http failure response for api/summary: 0 Unknown Error

      87 |     });
      88 | 
    > 89 |     it('pool data filtering is called', () => {
         |     ^
      90 |       const filterNonPoolDataSpy = spyOn(
      91 |         poolDetailsComponent,
      92 |         'filterNonPoolData'

      at new Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:116:22)
      at src/app/ceph/pool/pool-details/pool-details.component.spec.ts:89:5
      at ZoneDelegate.Object.<anonymous>.ZoneDelegate.invoke (node_modules/zone.js/dist/zone.js:391:26)
      at Zone.Object.<anonymous>.Zone.run (node_modules/zone.js/dist/zone.js:150:43)
      at src/app/ceph/pool/pool-details/pool-details.component.spec.ts:36:3
      at ZoneDelegate.Object.<anonymous>.ZoneDelegate.invoke (node_modules/zone.js/dist/zone.js:391:26)
      at Zone.Object.<anonymous>.Zone.run (node_modules/zone.js/dist/zone.js:150:43)
      at Object.<anonymous> (src/app/ceph/pool/pool-details/pool-details.component.spec.ts:12:1)

........................................................................................................................................

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ceph-dashboard@0.0.0 test:ci: `npm run test:config && JEST_SILENT_REPORTER_DOTS=true jest --coverage --reporters jest-silent-reporter`
npm ERR! Exit status 1
```

File: doc/conf.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 pygments_style = 'sphinx'
 
+html_file_suffix = ''
```
Comment (dillaman) [inline doc/conf.py]: Hmm -- on my Fedora 31 box, this change results in *all* the HTML files being generated w/o the `.html` suffix (and therefore no longer served from `admin/serve-docs`

Comment (badone) [reply doc/conf.py]: Yes, unfortunately I noticed this too yesterday on fedora and Bionic

Comment (dillaman) [reply doc/conf.py]: Fixed by #33562

**Pull request merged**